docs/versioned-plugins/filters/jdbc_streaming-v1.0.4.asciidoc

```diff
@@ -0,0 +1,226 @@
+:plugin: jdbc_streaming
+:type: filter
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: v1.0.4
+:release_date: 2018-04-02
+:changelog_url: https://github.com/logstash-plugins/logstash-filter-jdbc_streaming/blob/v1.0.4/CHANGELOG.md
+:include_path: ../include/6.x
+///////////////////////////////////////////
+END - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+
+[id="{version}-plugins-{type}s-{plugin}"]
+
+=== Jdbc_streaming filter plugin {version}
+
+include::{include_path}/plugin_header.asciidoc[]
+
+==== Description
+
+This filter executes a SQL query and store the result set in the field
+specified as `target`.
+It will cache the results locally in an LRU cache with expiry
+
+For example you can load a row based on an id from in the event
+
+[source,ruby]
+filter {
+  jdbc_streaming {
+    jdbc_driver_library => "/path/to/mysql-connector-java-5.1.34-bin.jar"
+    jdbc_driver_class => "com.mysql.jdbc.Driver"
+    jdbc_connection_string => ""jdbc:mysql://localhost:3306/mydatabase"
+    jdbc_user => "me"
+    jdbc_password => "secret"
+    statement => "select * from WORLD.COUNTRY WHERE Code = :code"
+    parameters => { "code" => "country_code"}
+    target => "country_details"
+  }
+}
+
+
+[id="{version}-plugins-{type}s-{plugin}-options"]
+==== Jdbc_streaming Filter Configuration Options
+
+This plugin supports the following configuration options plus the <<{version}-plugins-{type}s-{plugin}-common-options>> described later.
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Setting |Input type|Required
+| <<{version}-plugins-{type}s-{plugin}-cache_expiration>> |<<number,number>>|No
+| <<{version}-plugins-{type}s-{plugin}-cache_size>> |<<number,number>>|No
+| <<{version}-plugins-{type}s-{plugin}-default_hash>> |<<hash,hash>>|No
```

Comment:
@jsvd The doc build fails with errors because the type links like `<<hash,hash>>` don't resolve. They need to be `{logstash-ref}/configuration-file-structure.html#hash[hash]` for the versioned plugin reference.

Maybe you forgot to run the script with the  --repair option specified?